x-pack/plugins/apm/public/components/shared/Links/url_helpers.ts

```diff
@@ -52,9 +51,3 @@ export function legacyEncodeURIComponent(rawUrl?: string) {
 export function legacyDecodeURIComponent(encodedUrl?: string) {
   return encodedUrl && decodeURIComponent(encodedUrl.replace(/~/g, '%'));
 }
-
-// Make history singleton available across APM project.
-// This is not great. Other options are to use context or withRouter helper
-// React Context API is unstable and will change soon-ish (probably 16.3)
-// withRouter helper from react-router overrides several props (eg. `location`) which makes it less desireable
-export const history = createHistory();
```

Comment:
Thanks for removing this! It was ugly and probably not needed.
How do we import `history` going forward? Static imports, `useHistory` hook or something else?

Reply:
history is still a static singleton, just imported from `utils/history` now to keep it separate from the Links folder.